templates/download/server/index.html

```diff
@@ -46,23 +46,125 @@
     </div>
 </section>
 
-<section class="row">
-    <div class="strip-inner-wrapper equal-height--vertical-divider">
-        <div class="six-col equal-height--vertical-divider__item">
-            <h2>Server Provisioning</h2>
-            <h3>MAAS<h3>
-            <p>Metal as a Service (MAAS) offers cloud style provisioning for physical servers.</p>
-            <p>Designed for building data centres with complex networks.</p>
-            <p>Supports Windows, Ubuntu, CentOS, RHEL and SUSE.</p>
-            <p><a href="/download/server/provisioning">Install MAAS&nbsp;&rsaquo;</a></p>
+<section class="row no-border">
+    <div class="strip-inner-wrapper">
+        <div class="eight-col">
+            <h2>Your next steps with Ubuntu Server</h2>
+            <p>
+                Use Ubuntu&rsquo;s tools to help you provision and manage
+                your servers
+            </p>
+        </div>
+        <div class="equal-height">
+            <div class="four-col box">
+                <h3>
+                    Securely patch your kernels without rebooting with the
```

Comment:
This live patch item should be removed as per copy doc https://docs.google.com/a/canonical.com/document/d/1FFQQUtzFGH88hA30qnZm-3pwVqcVl9UYyrNT4qQWrSM/edit?pli=1&disco=AAAABIz7uXU